src/cmd/openmldb.cc

```diff
@@ -4435,14 +4393,6 @@ void StartClient() {
             HandleClientShowSchema(parts, &client);
         } else if (parts[0] == "getfollower") {
             HandleClientGetFollower(parts, &client);
-        } else if (parts[0] == "benput") {
-            HandleClientBenPut(parts, &client);
-        } else if (parts[0] == "benscan") {
-            HandleClientBenScan(parts, &client);
-        } else if (parts[0] == "benget") {
-            HandleClientBenGet(parts, &client);
-        } else if (parts[0] == "benchmark") {
-            HandleClientBenchmark(&client);
```

Comment:
Hi, @nautaa thanks for your pr.  
The implementation of these functions also needs to be deleted together.

Reply:
@dl239 Done.